app/services/create_new_device_alert.rb

```diff
@@ -0,0 +1,33 @@
+class CreateNewDeviceAlert < ApplicationJob
```

Comment:
I think you'll want to rebase or merge main to bring in #10342, which requires `frozen_string_literal` for every Ruby file.

```suggestion
frozen_string_literal: true

class CreateNewDeviceAlert < ApplicationJob
```

Reply:
Added that in. I'm I understanding correctly that frozen_string_literal is optional for .spec.rb files?

Reply:
Yeah, it's optional.

Related: https://github.com/18F/identity-idp/pull/10342#discussion_r1549835513